independent-publisher-connectors/Cloverly/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2266 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cloverly",
+    "description": "Cloverly's powerful API calculates the impact of common carbon-intensive activities in real time and uses verified, quality carbon offset projects to neutralize them. Businesses and consumers count on Cloverly to offset their carbon footprint and create positive environmental change based on real-time business activities.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api-prod-no-cert.cloverly.com",
+  "basePath": "/2021-10",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{transaction}/carbon": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "transaction_id": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Transaction Id"
+                },
+                "transaction_state": {
+                  "type": "string",
+                  "description": "",

```

Comment:
@troystaylor Empty property values can be removed until unless you have a specific reason to keep them.

Applies the same comment to all the similar empty property values.